src/Employer/Employer.Web/Controllers/ErrorController.cs

```diff
@@ -60,7 +60,7 @@ public async Task<IActionResult> ErrorHandler()
                 if (exception is InvalidStateException)
                 {
                     _logger.LogError(exception, "Exception on path: {route}", routeWhereExceptionOccurred);
-                    TempData.Add(TempDataKeys.DashboardErrorMessage, exception.Message);
+                    TempData[TempDataKeys.DashboardErrorMessage] = exception.Message;

```

Comment:
Is this hiding an issue? Could you log out what message is already there before overwriting so we can find the root issue. If it is multiple requests then something else has gone wrong.